src/sa_web/static/js/views/map-view.js

```diff
@@ -40,12 +43,25 @@ var Shareabouts = Shareabouts || {};
         } else if (config.type && config.type === 'landmark') {
           collectionId = config.id;
           self.layers[collectionId] = L.layerGroup();
-          self.landmarkLayerViews[collectionId] = {};
+
+          if (config.visibleDefault) {
+            self.map.addLayer(self.layers[collectionId]);
+          }
+          // REFACTOR
+          // add landmark layer views to the layerViews object:
+          self.layerViews[collectionId] = {};
 
           // Bind our landmark data events:
-          collection = self.options.landmarkCollections[collectionId];
-          collection.on('add', self.addLandmarkLayerView(collectionId), self);
-          collection.on('remove', self.removeLandmarkLayerView(collectionId), self);
+          //collection = self.options.landmarkCollections[collectionId];
+          
+          // filter out places collection from meta-collection object
```

Comment:
We should be able to fetch the landmark collection directly, using the id from the config (`collectionId`). I believe this would look like `collection = self.options.collection[collectionId];` That should eliminate the need for the filter and foreach loop below